WalletWasabi.Fluent/QRCodeDecoder/QRDecoder.cs

```diff
@@ -804,7 +804,7 @@ internal bool RemoveUnusedAlignMarks()
 		}
 
 		// test finder signature 1 1 3 1 1
-		internal bool TestFinderSig(int[] pos, int[] len, int index, out double module)
+		internal static bool TestFinderSig(int[] pos, int[] len, int index, out double module)
```

Comment:
Why should these be static?

Reply:
Take a loot at this https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1822#rule-description